source/fundamentals/crud/read-operations/skip.txt

```diff
@@ -0,0 +1,82 @@
+=====================
+Skip Returned Results
+=====================
+
+.. default-domain:: mongodb
+
+In this guide, we show you how to skip returned results.
+
+You can skip results using the ``skip()`` method of a 
+:java-docs:`FindIterable </apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#skip(int)>`
```

Comment:
Some users might not know what a `FindIterable` is, so we should provide a higher-level explanation. 
E.g. "You can specify the number of results to skip by calling the skip() method on the result of a query or in an aggregate pipeline."
Then, you can introduce the FindIterable as the result of the find() query on line 18 and something similar on line 24.


, so it could be better to specify the exact types and link to API documentation in the later.